operators/argo-controller/tox.ini

```diff
@@ -12,10 +12,19 @@ deps =
 	-rrequirements.txt
 
 [testenv:unit]
-commands = 
-	pytest test/unit
+commands =
+	pytest -v {toxinidir}/tests/unit
 
-[testenv:lint]
+[testenv:integration]
```

Comment:
This is out of scope for this PR, but integration tests by definition seem like they should reside at the root of the source tree, instead of in a single charm.

Reply:
I guess there's different levels of integration to work with.  The ones in the argo-controller are argo-controller specific (needs minio in order to turn on), but in some of the other bundles it is integration across charms in the repo.  Guess we need a convention that works nicely for both